Wordle.Api/Wordle.Api/Identity/IdentitySeed.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Wordle.Api.Data;
+
+namespace Wordle.Api.Identity;
+public static class IdentitySeed
+{
+    public static async Task SeedAsync(UserManager<AppUser> userManager, RoleManager<IdentityRole> roleManager)
+    {
+        // Seed Roles
+        await SeedRolesAsync(roleManager);
+
+        // Seed Admin User
+        await SeedAdminUserAsync(userManager);
+    }
+
+    private static async Task SeedRolesAsync(RoleManager<IdentityRole> roleManager)
+    {
+        // Seed Roles
+        if (!await roleManager.RoleExistsAsync(Roles.Admin))
+        {
+            await roleManager.CreateAsync(new IdentityRole(Roles.Admin));
+        }
+        if (!await roleManager.RoleExistsAsync(Roles.Special))
+        {
+            await roleManager.CreateAsync(new IdentityRole(Roles.Special));
+        }
+    }
+
+    private static async Task SeedAdminUserAsync(UserManager<AppUser> userManager)
+    {
+        // Seed Admin User
+        if (await userManager.FindByEmailAsync("Admin@intellitect.com") == null)
+        {
+            AppUser user = new AppUser
+            {
+                UserName = "Admin@intellitect.com",
+                Email = "Admin@intellitect.com",
+                Name = "Admin",
```

Comment:
it might be good to give this default admin master status, and have them be over 21